prow/jobs/governance.yaml

```diff
@@ -230,6 +230,34 @@ presubmits: # runs on PRs
             effect: NoSchedule
         nodeSelector:
             dedicated: "high-cpu"
+    - name: pull-kyma-crd-governance
+      annotations:
+        pipeline.trigger: "pr-submit"
+      labels:
+        prow.k8s.io/pubsub.project: "sap-kyma-prow"
+        prow.k8s.io/pubsub.runID: "pull-kyma-crd-governance"
+        prow.k8s.io/pubsub.topic: "prowjobs"
+        preset-build-pr: "true"
+        preset-dind-enabled: "true"
```

Comment:
I believe you also don't need those labels. Please remove them, so your ProwJob is not cluttered with additional env variables or volume mounts.